### DISCUSSION TITLE
SFR dimension warning

### DISCUSSION THREAD

**Discussion by hntig** [modflowpy/flopy#2046] [Q&A]
Hello. 

I tried running one of my old scripts in FloPy 3.5.0, and I get the following warning when creating my SFR package (multiple warnings in fact):

`WARNING: Unable to resolve dimension of ('gwf6', 'sfr', 'connectiondata', 'connectiondata', 'ic') based on shape "nconifno".`

The script was developed in Flopy 3.4.3 and worked perfectly fine before upgrading to 3.5.0. I also noticed that I don't get the warning if I don't use any connection data. Did something change in the new version or am I misunderstanding something?

My connection data are in the form of `connectiondata = [ [reach,  upstream_reach, -downstream_reach] ]` :

![conndata](https://github.com/modflowpy/flopy/assets/99178328/b9aafd62-5eb7-4991-b154-41299cf658ae)

**Comment by mwtoews** [comment]
The one big change for 3.5.0 is that SFR reach numbers were renamed `rno` to `ifno`. To check if this is a bug, see if you can update https://github.com/modflowpy/flopy/blob/02a2f91e802185fd6b31929f99e31fecbb41c499/flopy/mf6/coordinates/modeldimensions.py#L595
```diff
-                            or item[0] == "nconrno"
+                            or item[0] == "nconifno"
```
and re-run your script to see if it is resolved.

**Reply by wpbonelli** [reply]
This should have been mentioned in the [release notes](https://github.com/modflowpy/flopy/releases/tag/3.5.0), they have been updated.

It was [done](https://github.com/modflowpy/flopy/pull/1956/files#diff-672617a5df26d7dea3dc190e70e8baf223f0a00c81eaf0c52a3bc83c0779eb4e) to accommodate corresponding [`ifno`](https://github.com/MODFLOW-USGS/modflow6/pull/1361) [changes](https://github.com/MODFLOW-USGS/modflow6/pull/1360) [in](https://github.com/MODFLOW-USGS/modflow6/pull/1359) [mf6](https://github.com/MODFLOW-USGS/modflow6/pull/1356) which haven't made it into a release yet. But it seems ideal to support the old names for back-compatibility anyway
